superwise/assets/dashboards/superwise.json

```diff
@@ -0,0 +1 @@
query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:superwise.metric.overall.data_drift{*} by {model_name}","aggregator":"avg"}],"conditional_formats":[{"comparator":">","palette":"white_on_green","value":1}],"formulas":[{"formula":"count_not_null(query1)"}]}],"autoscale":true,"precision":2},"layout":{"x":5,"y":0,"width":2,"height":3}},{"id":2728068484701532,"definition":{"title":"Total # of Predictions","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"response_format":"scalar","queries":[{"query":"sum:superwise.metric.overall.quantity{*}","data_source":"metrics","name":"query1","aggregator":"sum"}]}],"autoscale":true,"precision":0},"layout":{"x":7,"y":0,"width":5,"height":3}},{"id":7223627866769632,"definition":{"title":"Incidents - Trend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"alias":"Models with Incidents","formula":"query1"}],"response_format":"timeseries","queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[]}],"style":{"palette":"purple","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":3,"width":4,"height":3}},{"id":1954347328001564,"definition":{"title":"Total # of Predictions - Trend","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"response_format":"timeseries","queries":[{"query":"max:superwise.metric.overall.quantity{*} by {model_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"area"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":3,"width":4,"height":3}},{"id":7366254108548012,"definition":{"title":"Models Input Drift From Baseline","title_size":"16","title_align":"left","time":{},"type":"toplist","requests":[{"formulas":[{"formula":"query1","limit":{"count":5,"order":"desc"}}],"conditional_formats":[{"comparator":"<","palette":"white_on_green","value":50},{"comparator":">=","palette":"white_on_red","value":80},{"comparator":">=","palette":"white_on_yellow","value":50}],"response_format":"scalar","queries":[{"query":"max:superwise.metric.overall.data_drift{*} by {model_name}","data_source":"metrics","name":"query1","aggregator":"last"}]}]},"layout":{"x":8,"y":3,"width":4,"height":3}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"3rs-pu9-3be"}
```

Comment:
Can you update the description of the dashboard?

Reply:
Fix